plane/source/docs/roll-pitch-controller-tuning.rst

```diff
@@ -18,11 +18,10 @@ new aircraft, as everything else relies on getting this right.
 
    The default values for the
    roll and pitch controllers in Plane are quite deliberately too small for
-   most aircraft. This is because small values will cause APM to not
+   most aircraft. This is because small values will cause autopilot to not
    navigate well and be sluggish, but are less likely to cause the aircraft
-   to crash. The defaults also have a zero 'I' gain, as you should not
-   introduce any 'I' (integrator) until you have correctly tuned the 'P'
-   gain.
+   to crash. The pitch defaults also have a large 'I' gain, to try to compensate for poor initial CG location.
+   Once trimmed via :ref:`SERVO_AUTO_TRIM<SERVO_AUTO_TRIM>`, this value should be zero'd and full tuning done, as instructed below.
```

Comment:
the I should not be set to zero. Usually the ratio between P and I should be about 5:1 (so P about 5x the value of I)

Reply:
done